check-stylish.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+function fail_maybe_explain {
+    if [ -n "$TRAVIS" ];
+    then
+        echo "Run ./check-stylish.sh locally to stylize your files / debug this"
+    fi
+    exit 1
+}
+
+if [ -n "$TRAVIS" ];
+then
+    # It takes a while to compile, let's not tax Travis' infrastructure for no
+    # reason. We can't cache it via cabsnap because that's based on the install-
+    # plan for Idris, which of course doesn't have stylish-haskell as a
+    # dependency.
+    STYLISH_EXE=$(mktemp)
+    wget -O $STYLISH_EXE "https://s3-us-west-2.amazonaws.com/idris-travis-binaries/stylish-haskell"
```

Comment:
OK, so I think that if it's impractical to `stack install stylish-haskell` and cache the Stack cache, then the next best thing is storing the binary on idris-lang.org somewhere. I don't want your personal s3 account to be a failure point for Idris's CI.